tox.ini

```diff
@@ -21,38 +21,35 @@ deps =
 
 [testenv:pre-commit]
 deps =
-    torch @ https://download.pytorch.org/whl/lts/1.8/cu111/torch-1.8.2%2Bcu111-cp38-cp38-linux_x86_64.whl
-    torchvision @ https://download.pytorch.org/whl/lts/1.8/cu111/torchvision-0.9.2%2Bcu111-cp38-cp38-linux_x86_64.whl
-    torchtext @ https://download.pytorch.org/whl/lts/1.8/torchtext-0.9.2-cp38-cp38-linux_x86_64.whl
+    torch @ https://download.pytorch.org/whl/cu116/torch-1.13.1%2Bcu116-cp38-cp38-linux_x86_64.whl
+    torchvision @ https://download.pytorch.org/whl/cu116/torchvision-0.14.1%2Bcu116-cp38-cp38-linux_x86_64.whl
+    torchtext @ https://download.pytorch.org/whl/torchtext-0.14.1-cp38-cp38-linux_x86_64.whl
     -r{toxinidir}/requirements/dev.txt
     -r{toxinidir}/requirements/base.txt
-    -r{toxinidir}/requirements/anomaly.txt
+    # -r{toxinidir}/requirements/anomaly.txt
     -r{toxinidir}/requirements/openvino.txt
-commands = 
+use_develop = true
+commands =
     pip install -e .[mpa]
     pre-commit run --all-files
 
 
 [testenv:pre-merge]
 deps =
-    torch @ https://download.pytorch.org/whl/lts/1.8/cu111/torch-1.8.2%2Bcu111-cp38-cp38-linux_x86_64.whl
-    torchvision @ https://download.pytorch.org/whl/lts/1.8/cu111/torchvision-0.9.2%2Bcu111-cp38-cp38-linux_x86_64.whl
-    torchtext @ https://download.pytorch.org/whl/lts/1.8/torchtext-0.9.2-cp38-cp38-linux_x86_64.whl
-    mmcv-full @ https://download.openmmlab.com/mmcv/dist/cu111/torch1.8.0/mmcv_full-1.6.1-cp38-cp38-manylinux1_x86_64.whl
+    torch @ https://download.pytorch.org/whl/cu116/torch-1.13.1%2Bcu116-cp38-cp38-linux_x86_64.whl
+    torchvision @ https://download.pytorch.org/whl/cu116/torchvision-0.14.1%2Bcu116-cp38-cp38-linux_x86_64.whl
+    torchtext @ https://download.pytorch.org/whl/torchtext-0.14.1-cp38-cp38-linux_x86_64.whl
     coverage
     pytest-timeout
     pytest-mock
     -r{toxinidir}/requirements/base.txt
-    -r{toxinidir}/requirements/anomaly.txt
+    # -r{toxinidir}/requirements/anomaly.txt
```

Comment:
@ashwinvaidya17 Temporarily disabled this requirement as anomalib has torch version conflict. In this PR, all anomaly tests are failing due to absence of anomalib. What do you think? Would you create a PR to this branch? Or, will it be better to merge this PR and you create an update PR after that?